<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBOp_Rechnung</class>
 <widget class="QDialog" name="DBOp_Rechnung">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rechnung</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>528</width>
     <height>252</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelRechnungsID">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>RechnungsID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineRechnungsID">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelRechnungsdatum">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Rechnungsdatum</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineRechnungsdatum">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelFaelligkeit">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Fälligkeit</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineFaelligkeit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelMwSt">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MwSt.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDoubleSpinBox" name="doubleMwSt">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <double>99999.990000000005239</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEuro">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Sans Serif</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>€</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelBestellungsID">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>BestellungsID</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBestellungsID">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushNeuB">
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Neu...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelZahlungsart">
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Zahlungsart</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboZahlungsart">
        <item>
         <property name="text">
          <string>Bankeinzug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rechnung</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>526</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DBOp_Rechnung</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DBOp_Rechnung</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
